server/routes/pubDocument.tsx

```diff
@@ -291,3 +305,64 @@ app.get(
 		return renderPubDocument(res, pubData, initialData, customScripts);
 	}),
 );
+
+app.get(
```

Comment:
⚠️ I'm kind of iffy about adding two new routes for this purpose. The user already has to supply `commentHash` in the URL in order to be able to make comments as a logged-out user. Shouldn't we be able to infer that they're in "visiting commenter" mode by that alone?

Reply:
I'm going to solve this part when we get back, but I completely agree!